app/components/navbar/navbar.js

```diff
@@ -36,6 +36,7 @@ var Navbar = React.createClass({
     getUploadUrl: React.PropTypes.func,
     onLogout: React.PropTypes.func,
     trackMetric: React.PropTypes.func.isRequired,
+    permsOfClinician: React.PropTypes.object,
```

Comment:
Recommend that we rename `permsOfClinician` (here and in all other instances) to `permsOfLoggedInUser`, as this also applies to non-clinician care team members.  

This prop name is already in use extensively in the app, so we should keep it consistent.